### PR TITLE
Adding wired buttons

### PR DESCRIPTION
**Pull request by 4var1**

### CODE REVIEW

**Comment by 4var1** [comment]
I tried removing the line from Makefile about key manager as you suggested - but then it wouldn't build?

File: EBike_wireless_TSDZ2/firmware/Makefile [hunk 1]
```diff
@@ -317,7 +322,7 @@ flash: default
 erase:
 	nrfjprog -f nrf52 --eraseall
 
-SDK_CONFIG_FILE := ../config/sdk_config.h
+SDK_CONFIG_FILE := $(PROJ_DIR)/include/sdk_config.h
```
Comment (casainho) [inline EBike_wireless_TSDZ2/firmware/Makefile:325]: Why did you change this line?

**Comment by casainho** [comment]
can you look at my previous comment?

File: EBike_wireless_TSDZ2/firmware/Makefile (hunk 1 above)
Comment (4var1) [reply EBike_wireless_TSDZ2/firmware/Makefile:325]: Because I'm going to add LED sequences - you need to edit lines in sdk_config.h to enable pwm_led and pwm_low power - rananna had done the same. There also isn't a config folder in the source - so I assumed I needed to refer to the one I'd edited in \include. Is \config a folder I should be creating for myself?

Comment (4var1) [reply EBike_wireless_TSDZ2/firmware/Makefile:325]: I added LED support as I was just trying out disp_soc() on motor power on start. But the code copied from the remote blocks the thread and so motor init times out with error brakes. I'm planning to implement a non-blocking LED sequence player so that we can have LED sequences showing and not hold up other processing.

Comment (4var1) [reply EBike_wireless_TSDZ2/firmware/Makefile:325]: I now realise I should try things out on a separate branch and merge back in yes?... All part of a steep learning curve!

Comment (casainho) [reply EBike_wireless_TSDZ2/firmware/Makefile:325]: Well, I see @rananna does various commits to the same PR. Maybe you can ask him help on this.

Ok about the sdk_config.h.

File: EBike_wireless_TSDZ2/firmware/include/screen.h [hunk 2]
```diff
@@ -576,7 +576,7 @@ extern volatile bool g_graphs_ui_update[3];
 #define _S(longstr, shortstr) shortstr
 #endif
 
-#if defined(SW102) || defined(DISPLAY_860C)
+#if defined(SW102) || defined(DISPLAY_860C) || defined(BOARD_PCA10059)
```
Comment (casainho) [inline EBike_wireless_TSDZ2/firmware/include/screen.h:579]: Please remove all references to the displays.

Comment (4var1) [reply EBike_wireless_TSDZ2/firmware/include/screen.h:579]: Ok - my thinking was to make it compatible with the older codebase so that if we made improvements they could be merged back in. So we can just cut out all the #defines for any other devices than the blue dongle?

Comment (casainho) [reply EBike_wireless_TSDZ2/firmware/include/screen.h:579]: As I told before, I do not want to touch on the displays firmware. The life is so short that I think the future is probably only this wireless Ebike. Please remove references to them.
